src/app/modules/test-step-selector/test-step-tree-node.ts

```diff
@@ -8,8 +8,9 @@ export class TestStepTreeNode implements TreeNode {
   collapsedCssClasses = 'category fas fa-caret-right';
   expandedCssClasses = 'category fas fa-caret-down';
   leafCssClasses: string;
+  public root: TestStepTreeNode;
 
-  constructor(private testStepTree: TestStepNode) {
+  constructor(private testStepTree: TestStepNode, root: TestStepTreeNode) {
```

Comment:
same as in test-navigator, _consider_ making this an optional parameter, and / or provide a default value. Or leave as is if that would make things intransparent.